save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/User.kt

```diff
@@ -14,4 +15,14 @@ class User(
     var password: String?,
     var role: String?,
     var source: String,
-) : BaseEntity()
+) : BaseEntity() {
+    /**
+     * @param projects
+     * @return [UserDto] object
+     */
+    fun toDto(projects: Map<String, Role?> = emptyMap()) = UserDto(
+        name = name,
+        source = source,
+        projects = projects.toMutableMap(),
```

Comment:
why do you need mutable map here, btw? 
Any possibilities to avoid it?

Reply:
I used to need it but now I am free to remove it.